### ISSUE TITLE
Create option for component documentation to have horizontal navigation

### ISSUE THREAD

**Issue by kimadactyl** [devnacho/mountain_view#58]
It would be great if the left hand navigation could go at the top instead to make testing responsive components easier. Obviously can hide it using `display: none` but be great to not have to do this :)

**Comment by kimadactyl** [comment]
Yeah can possibly do a PR, would be some considerable yak shaving for where I'm at right now but can have a look soon :smile:

**Comment by kitop** [comment]
Sure, thanks!

If you want, maybe we can start with screenshots of what you have first (covering sensitive information if needed), so we are all in the same page regarding looks and responsiveness.

**Comment by Spone** [comment]
My 2 cents: you can also keep the navigation vertical (which is the most effective when you have a lot of components) but add a toggle button to hide it, so you can display your component in full width. That's the solution we usually favor on our styleguides.
